evm/rlp/headers.py

```diff
@@ -190,15 +197,33 @@ def __repr__(self):
 
     @property
     def hash(self):
-        return keccak(rlp.encode(self))
+        int_to_bytes32 = compose(
+            pad32,
+            int_to_big_endian,
+        )
+        header_hash = keccak(
+            b''.join(
+                (
```

Comment:
*whitespace nitpick*

This should be a bit easier to read if you write this as:

```python
b''.join((
    thing_a,
    thing_b,
))
```

Reply:
You're right. I will change it.